python/ray/services.py

```diff
@@ -21,13 +21,14 @@
 import ray.ray_constants as ray_constants
 
 from ray.tempfile_services import (get_ipython_notebook_path, get_temp_root,
-                                   new_redis_log_file)
+                                   new_redis_log_file, get_logs_dir_path)
```

Comment:
nit, let's add comma after the last item, so they will be formatted as item per line.